troubleshooting/application-title-in-email-templates-wrong.md

```diff
@@ -0,0 +1,55 @@
+---
+title: &#123;&#123;Application.Title&#125;&#125; in Email Template is Wrong
+description: You renamed the name of your project in Telerik Platform portal, but in emails &#123;&#123;Application.Title&#125;&#125; still the old name appears.
+type: troubleshooting
+page_title: How to Change the &#123;&#123;Application.Title&#125;&#125; in Email Templates
+slug: application-title-in-email-templates-wrong
+position:
+tags: backendservices, portal, email-notification
+teampulseid:
+ticketid: 1111828, 1072170, 1060801, 1054819, 1087785, 1100004, 1058035, 1087785
+pitsid:
+
+---
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.tp}}, {{site.bs}} {{site.bs-n}}</td>
+  </tr>
+</table>
+
+## Description
+After changing the application name in Telerik Platform portal, the old name {{Application.Title}} still appearing in all automated emails.
+
+## Cause
+Internal limitation. {{Application.Title}} gets its value not from the application name but from another field (backend project name) which is not changed when you change the appliaction name from the portal UI.

```

Comment:
Just a suggestion for different wording:
Internal limitation. The {{Application.Title}} value is not based on the application name field, but instead takes/gets its value from a different field (backend project name). The value in this field remains unchanged when you modify the application name from the portal UI.